packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -103,9 +106,15 @@ export const iconMetadata: {
     IconBellAndComment: {
         tags: ['alarm', 'notification']
     },
+    IconBellCheck: {
+        tags: ['acknowledged', 'alarm']
```

Comment:
@fredvisser this change is already merged but @m-akinc's proposal I thought was really clear. That the acknowledged state should be a visual of the alarm is still active but it has a check to show the active alarm is acknowledged.

![image](https://github.com/user-attachments/assets/96c416e6-b5e3-4437-8f45-676e0727590c)

Did we already discuss it with @NIbokeefe and decide to instead have the visual shown here where the alarm does not look active and has a check?

Reply:
(and I know my proposal was not popular compared to Mert's but I think consistent bell usage is even clearer)
![image](https://github.com/user-attachments/assets/c71c7655-eb5e-4ead-9da2-733bd5762d7e)